.vib/osclass/goss/vars.yaml

```diff
@@ -0,0 +1,18 @@
+containerPorts:
+  http: 8081
+service:
+  ports:
+    https: 443
```

Comment:
You have updated the tests but not changed the port here

Reply:
This keeps failing when changing the port to a non default one:
```
<testcase name="HTTP https://osclass:444 status" time="5.000">
<system-err>https://osclass:444: status: Error: Get https://osclass:444: net/http: request canceled (Client.Timeout exceeded while awaiting headers)</system-err>
<failure>https://osclass:444: status: Error: Get https://osclass:444: net/http: request canceled (Client.Timeout exceeded while awaiting headers)</failure>
</testcase>
```

Reply:
I've added a timeout of 70000 and it works now

Reply:
@mdhont Is such a huge timeout necessary? The almost 12 minutes used seems a bit overboard for me. Have you encountered instances where the tests got close to that length? From what I recall, local testing ranged between 20-60s for the test execution.